api/graphql/schema.graphql

```diff
@@ -1342,14 +1350,17 @@ input CreatePoolCandidateSearchRequestInput {
   jobTitle: String! @rename(attribute: "job_title")
   additionalComments: String @rename(attribute: "additional_comments")
   applicantFilter: ApplicantFilterBelongsTo!
-  # requestedDate will be set to current date in Eloquent model.
-  # status will be set to PENDING in Eloquent model.
+  # status will be set to NEW by default in Postgres
   wasEmpty: Boolean @rename(attribute: "was_empty")
+  managerJobTitle: String! @rename(attribute: "manager_job_title")
+  positionType: PoolCandidateSearchPositionType!
+    @rename(attribute: "position_type")
 }
 
 input UpdatePoolCandidateSearchRequestInput {
   adminNotes: String @rename(attribute: "admin_notes")
   status: PoolCandidateSearchStatus
+  # statusChangedAt updated in the Eloquent model
```

Comment:
`status` now is set to "NEW" by default, `requestedDate` is just `created_date` which I think is made clear in the type definition and the comment is confusing with it referencing Eloquent.